posix.mak

```
@@ -29,7 +29,12 @@ INSTALL_DIR=../install
 DOCDIR=doc
 IMPDIR=import
 
-OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
+# -fPIC is enabled by default and can be disabled with PIC=0
+ifneq ($(PIC),0)
+    DEFAULT_PIC:=-fPIC
```

`DEFAULT_PIC` seems misleading to me, how about `PIC_FLAG`?